.github/wiki/Project-Configurations.md

```diff
@@ -0,0 +1,136 @@
+## Project Configurations
+
+This document presents in detail the set of configurations used in the iOS projects developed at Nimble. Included in this document:
+
+- Terminologies and topics related to project configurations.
+- Basic targets, schemes, build settings options, etc., for building a project.
+
+### **Targets**
```

Comment:
Why does this have bold tag?
```suggestion
### Targets
```

Reply:
@blyscuit I copied from Notion :D, will recheck on another pages :D

Reply:
Fixed in https://github.com/nimblehq/ios-templates/pull/268/commits/4ac43d42bd69de88234f85aee75cd8736fdfc56f